stubs/docker-compose.yml

```diff
@@ -41,6 +42,19 @@ services:
             - 'sailmysql:/var/lib/mysql'
         networks:
             - sail
+#    pgsql:
+#        image: postgres:12
+#        ports:
+#            - '${FORWARD_DB_PORT:-5432}:5432'
+#        environment:
+#            PGPASSWORD: '${DB_PASSWORD}'
+#            POSTGRES_DATABASE: '${DB_DATABASE}'
```

Comment:
```suggestion
#            POSTGRES_DATABASE: '${DB_DATABASE}'
#            POSTGRES_DB: '${DB_DATABASE}'
```

as described on https://hub.docker.com/_/postgres

Reply:
I'm not sure if this is the right suggestion:

> If it is not specified, then the value of POSTGRES_USER will be used.

If we are talking about the default, maybe better suggestion is, what do you think?

```
POSTGRES_DB: 'postgres'
```

Reply:
That's right. Postgres Docker image uses `POSTGRES_DB`, not `POSTGRES_DATABASE` :)

I personally have default values like so:

```
    pgsql:
        image: 'postgres:12'
        ports:
            - '${FORWARD_DB_PORT:-5432}:5432'
        environment:
            POSTGRES_DB: '${DB_DATABASE}'
            POSTGRES_USER: '${DB_USERNAME}'
            POSTGRES_PASSWORD: '${DB_PASSWORD:-secret}'
            PGPASSWORD: '${DB_PASSWORD:-secret}'
        volumes:
            - 'sailpgsql:/var/lib/postgresql/data'
        networks:
            - sail
```

However, I noticed mysql container (just above) doesn't set default password, so maybe for consistency we shouldn't set them here either 🤔   Maybe @driesvints has an opinion on that?

Reply:
It looks PG doesn't allow empty password, while MySQL does. And the default `.env` ships with empty password.